index.js

```diff
@@ -93,7 +90,7 @@ addBook.addEventListener('click', (e) => {
   addBtn.innerText = 'Add';
   addBtn.id = 'btn';
   form.appendChild(addBtn);
-  allBooks = JSON.parse(localStorage.getItem('bookList'));
+  allBooks = JSON.parse(JSON.stringify(localStorage.getItem('book-List')));
```

Comment:
- [ ] As suggested by the previous reviewer [here](https://github.com/Yemisirach/Awsome-book-ES6/pull/1#discussion_r1087255864) kindly make the required change. I am too having the same error.

<img width="967" alt="Screenshot 2023-01-26 at 12 39 03 PM" src="https://user-images.githubusercontent.com/70269507/214804290-5e8a4eed-c192-46df-8995-de024dd80f0c.png">

Reply:
Hello Syeda there is no issue on add.js my side is working well. how can I fix?